### ISSUE TITLE
Add documentation for stats calculated by gt2popsum/gt.to.popsum

### ISSUE THREAD

**Issue by zkamvar** [knausb/vcfR#101]
The functions `gt2popsum()` and `gt.to.popsum()` appear to calculate different summaries:

`gt2popsum()`:

https://github.com/knausb/vcfR/blob/7fc3618b294a44cd732b852dea656ccd242ff789/R/proc_chromR.R#L384-L386

`gt.to.popsum()`:

https://github.com/knausb/vcfR/blob/7fc3618b294a44cd732b852dea656ccd242ff789/R/proc_chromR.R#L449

However, I can't seem to find documentation as to what these summaries are or how they are calculated. Could you include a description of these summaries in a future release?

Cheers,
Zhian

**Comment by knausb** [comment]
Hi Zhian,

The function `gt2popsum()` is old. It is a pure R implementation and I have now marked it for deprecation. The function `gt.to.popsum()` is its replacement and is a wrapper to compiled code. I've added some documentation for it and it is available on the master branch. If you'd like you can install this as follows.

```
devtools::install_github(repo="knausb/vcfR")
```

I have separated the documentation for `gt.to.popsum()` from `proc_chrom()` in a hope to simplify things a bit. Would you like to review these changes?

Thanks!
Brian
Would you

**Comment by zkamvar** [comment]
It looks good to me! It turns out I was interpreting `Ne` all wrong 🤭! Out of curiosity, what were the values calculated by the deprecated version? 

By the way, I like how you handled the deprecation by giving the users a clear stop sign, but allow them to roll through if they want to.

**Comment by knausb** [comment]
The other values included number of reference and alternate alleles (note that I assumed diploidy there which is part of the reason I deprecated it), Hardy-Weinberg, population mutation rates (thetas) and tests for selection (ratios of thetas). I'd like to get that back in to vcfR as either part of the function or a new function. But it was hurting performance for `proc_chrom()` and was not part of any default plotting, so I chose not to include them.
